pkg/config/config.go

```diff
@@ -403,7 +403,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("aggregator_use_tags_store", true)
 	config.BindEnvAndSetDefault("basic_telemetry_add_container_tags", false) // configure adding the agent container tags to the basic agent telemetry metrics (e.g. `datadog.agent.running`)
 	config.BindEnvAndSetDefault("aggregator_flush_metrics_and_serialize_in_parallel", true)
-	config.BindEnvAndSetDefault("aggregator_flush_metrics_and_serialize_in_parallel_chan_size", 500*1000) // This default value ensures the aggregator dont' wait for the serializer.
+	config.BindEnvAndSetDefault("aggregator_flush_metrics_and_serialize_in_parallel_chan_size", 2000)
+	config.BindEnvAndSetDefault("aggregator_flush_metrics_and_serialize_in_parallel_buffer_size", 4000)
```

Comment:
2000 buffers of 4000 elements means that potentially we can have 8M series in flight between the aggregator and the serializer. Not only it is higher than previously, it will also use over 1.4G of ram just buffering series (at 152 bytes per serie, plus points and buffer slices themselves). I'm not sure we want to buffer this much, even if it that otherwise we have to wait for the serializer in the aggregator. Do you think a smaller buffer and channel sizes would work?

Reply:
Good catch, I wanted the product to be 800 000. 
Note: This is limited by the number of series created during a flush and previously there was a slice which can contain much more elements.